### DISCUSSION TITLE
How does Keka handle corruption of one or more parts of a multi-part archive?

### DISCUSSION THREAD

**Discussion by alexclaydon** [aonez/Keka#1220] [Q&A]
Hello, this question really goes to the underlying compression library being used, I guess, but I'm just curious whether Keka is capable of unzipping a multi-part archive when one (or more) parts is corrupted or missing?  I'm researching a convenient solution for archiving a large-ish (300gb) volume of important personal data over several BDXL discs.  Thank you!

**Comment by aonez** [comment]
Hi @alexclaydon,

Keka can extract corrupted RAR files using the PAR2 thus recovering broken or missing volume parts. I think this will be a great option but you need to have a valid RAR license to make the compression. Keka [can compress in RAR format*](https://github.com/aonez/Keka/wiki/Rar-compression) but does not produce the PAR2 files, those you'll need to produce on your own.

Also LZIP has good recovery methods, but Keka is not prepared to handle the [recovery process](https://www.nongnu.org/lzip/manual/lziprecover_manual.html).
